packages/retail-ui-extensions/src/components/CameraScanner/CameraScanner.ts

```diff
@@ -8,14 +8,14 @@ export type BannerType =
   | 'Critical'
   | 'Default';
 
-export interface Information {
-  message: string;
-  bannerType?: BannerType;
+export interface CameraScannerBannerProps {
+  title: string;
+  variant?: BannerType;
```

Comment:
I believe this should be of type `BannerVariant` and then you can delete `BannerType`